Elasticsearch Update (#4062)
astronomer/docs: software/export-task-logs.md

Add Airflow task logs from your Astronomer Deployment to an existing Elasticsearch instance on [Elastic Cloud](https://www.elastic.co/cloud/) to centralize log management and analysis. Centralized log management allows you to quickly identify, troubleshoot, and resolve task failure issues. Although these examples use Elastic Cloud, you can also use AWS Managed OpenSearch Service or any other elastic service (managed or hosted). With an external Elasticsearch instance configured for Astronomer Software, you can see the logs in your Elasticsearch instance and browse the logs from the Software UI.

:::note

> If using an existing Elasticsearch instance, ensure that the index template is configured to enable auto creation of new indices. 

Reviewer: ```suggestion
If  you use an existing Elasticsearch instance, make sure that the index template is configured to enable auto creation of new indices. 
```

    ![Discover menu location](/img/software/elasticsearch-discover.png)

> 3. Enter `fluentd.*` or `vector.*` (depending on whether you are using Vector Sidecar Logging) in the **Name** field, enter `@timestamp` in the **Timestamp field**, and then click **Create index pattern**.

Reviewer: ```suggestion
3. Enter `fluentd.*`, or `vector.*` if you use Vector Sidecar Logging. In the **Name** field, enter `@timestamp` in the **Timestamp field**, and then click **Create index pattern**.
```
